docs/examples/koa-webhook-bot.js

```diff
@@ -23,8 +27,12 @@ bot.telegram.setWebhook('https://-----.localtunnel.me/secret-path')
 const app = new Koa()
 app.use(koaBody())
 // @ts-ignore
-app.use((ctx, next) => ctx.method === 'POST' || ctx.url === '/secret-path'
-  ? bot.handleUpdate(ctx.request.body)
-  : next()
-)
+app.use((ctx, next) => {
+  if (ctx.url === '/secret-path') {
```

Comment:
```suggestion
  if (ctx.method === 'POST' && ctx.url === '/secret-path') {
```

Reply:
not matters. when matching the secret path, delegate all methods to the handler.